lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -1670,7 +1670,7 @@ def test_certificate_generation_for_students(self):
             'failed': 3,
             'skipped': 2
         }
-        with self.assertNumQueries(175):
+        with self.assertNumQueries(191):
```

Comment:
This one is a bit concerning, but it makes sense. 2 extra transaction wrappers for 8 attempted certificate generations = 16 additional queries.